packages/language-service/src/locate_symbol.ts

```diff
@@ -20,6 +20,43 @@ export interface SymbolInfo {
   compileTypeSummary: CompileTypeSummary|undefined;
 }
 
+function getSymbolInMicrosyntax(
+    info: AstResult, templatePosition: number, path: TemplateAstPath, attribute: Attribute) {
+  if (!attribute.valueSpan) {
+    return;
+  }
+  let result: {symbol: Symbol, span: Span}|undefined;
+  const {templateBindings} = info.expressionParser.parseTemplateBindings(
+      attribute.name, attribute.value, attribute.sourceSpan.toString(),
+      attribute.valueSpan.start.offset);
+  // Find where the cursor is relative to the start of the attribute value.
+  const valueRelativePosition: number = templatePosition - attribute.valueSpan.start.offset;
+
+  // Find the symbol that contains the position.
+  templateBindings.forEach(tb => {
+    if (!tb.keyIsVar) {
+      if (tb.expression && inSpan(valueRelativePosition, tb.expression.ast.span)) {
```

Comment:
Now that we are on TS 3.7, you can do

```suggestion
      if (inSpan(valueRelativePosition, tb.expression?.ast?.span)) {
```